app/config.py

```diff
@@ -503,3 +503,11 @@ def setup_nameservers():
     raise RuntimeError(
         "Please define RECOVERY_CODE_HMAC_SECRET in your configuration with a random string at least 16 chars long"
     )
+
+# the minimum rspamd spam score above which emails that fail DMARC should be quarantined
+if "MIN_RSPAMD_SCORE_FOR_FAILED_DMARC" in os.environ:
+    MIN_RSPAMD_SCORE_FOR_FAILED_DMARC = float(
+        os.environ["MIN_RSPAMD_SCORE_FOR_FAILED_DMARC"]
+    )
+else:
+    MIN_RSPAMD_SCORE_FOR_FAILED_DMARC = None
```

Comment:
MIN_RSPAMD_SCORE_FOR_FAILED_DMARC = float(os.environ.get("MIN_RSPAMD_SCORE_FOR_FAILED_DMARC",0))

Reply:
This is different actually, I don't want to default `MIN_RSPAMD_SCORE_FOR_FAILED_DMARC` to 0 but None instead. The comparison with None is then used in `dmarc.py`

Reply:
If it's 0 by default and the comparison is `<` then it'will effectively be disabled won't it?

Reply:
I prefer to be explicit and not use 0 as default value.